cdap-app-fabric/src/main/java/io/cdap/cdap/internal/github/GitHubIO.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.internal.github;
+
+import com.google.gson.JsonObject;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+public class GitHubIO {
```

Comment:
add javadoc for public class
also GithubIO is a confusing name.
I think you'd better have GitHubCheckinRequest and GithubCheckoutRequest separately.